debian/xenial/kubeadm/debian/rules

```diff
@@ -8,10 +8,9 @@ build:
 
 binary:
 	mkdir -p usr/bin
-	#curl  --fail -sS -L \
-	#	-o usr/bin/kubeadm \
-	#	"https://storage.googleapis.com/kubernetes-release/release/v{{ .Version }}/bin/linux/{{ .Arch }}/kubeadm"
-	touch usr/bin/kubeadm
+	curl  --fail -sS -L \
+		-o usr/bin/kubeadm \
+		"https://storage.googleapis.com/kubeadm/v{{ .Version }}/bin/kubeadm"
```

Comment:
At the moment only I do, but I can add folks to it. However, this is only meant to be temporary. Also, all of this is TBD, I am not happy with how it's working right now, e.g. I want to be able to build from source...